packages.md

```diff
@@ -3,24 +3,28 @@
 
 | Package Id     | Readme    | Changelog                 | Published Url       |
 |----------------|-----------|---------------------------|---------------------|
-| [`Azure.Core`]( sdk/core/Azure.Core/src/Azure.Core.csproj )| [Readme](sdk/core/Azure.Core/README.md) |  N/A  | [![Azure.Core](https://img.shields.io/nuget/vpre/Azure.Core.svg)]( https://www.nuget.org/packages/Azure.Core/ ) |
-| [`Azure.Data.AppConfiguration`]( sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj )| [Readme](sdk/appconfiguration/Azure.Data.AppConfiguration/README.md) |  N/A  | [![Azure.Data.AppConfiguration](https://img.shields.io/nuget/vpre/Azure.Data.AppConfiguration.svg)]( https://www.nuget.org/packages/Azure.Data.AppConfiguration/ ) |
-| [`Azure.Identity`]( sdk/identity/Azure.Identity/src/Azure.Identity.csproj )| [Readme](sdk/identity/Azure.Identity/README.md) |  N/A  | [![Azure.Identity](https://img.shields.io/nuget/vpre/Azure.Identity.svg)]( https://www.nuget.org/packages/Azure.Identity/ ) |
+| [`Azure.AI.InkRecognizer`]( sdk/cognitiveservices/InkRecognizer/src/Azure.AI.InkRecognizer.csproj )|  N/A  |  N/A  | [![Azure.AI.InkRecognizer](https://img.shields.io/nuget/vpre/Azure.AI.InkRecognizer.svg)]( https://www.nuget.org/packages/Azure.AI.InkRecognizer/ ) |
```

Comment:
Please exclude the InkRecognizer packages. In fact would you mind submitting a PR to remove those from the repo.

Reply:
Removed `InkRecognizer.UWP.Stroke`, `InkRecognizer.WPF.Stroke` and `InkRecognizer` from the repo.